xbmc/cores/VideoPlayer/VideoPlayerAudio.cpp

```diff
@@ -199,9 +199,13 @@ void CVideoPlayerAudio::UpdatePlayerInfo()
   s << "aq:"     << std::setw(2) << std::min(99,m_messageQueue.GetLevel()) << "%";
   s << ", Kb/s:" << std::fixed << std::setprecision(2) << m_audioStats.GetBitrate() / 1024.0;
 
+  // print a/v discon adjustments counter when audio is in passthrough mode
+  if (m_synctype == SYNC_DISCON)
+    s << ", discon:" << m_disconAdjustCounter;
```

Comment:
discon is for "discontinuity"?
if so, since there is much space complete word is better understood

Reply:
Yes, I think it's this, but I'm not sure if that's the best word for what it stands for.

When the a/v adjustment is done by resampling the audio shows "rr:" resample ratio I imagine (the author liked short words 🙂).

When it is not possible to resample due passthrough enabled is adjusted in the video clock: generates one or more micro-pause or micro-acceleration on video to sync with audio.

Maybe better change to "sync adjust:" or "a/v adjust:"   ?

I want avoid long words as "discontinuity adjustments counter" 😛  as this debug info OSD only

Reply:
is not "detection" but "correction"

I personally would prefer a single short word like drop, skip, etc. "adjust" ?

a/v off-sync is in line 3 Player

Reply:
In any case, to find out what it is, people will have to look it up on the wiki.